plugin/src/test/resources/expected_output/basic/akka/actor/Identify.java

```diff
@@ -0,0 +1,20 @@
+package akka.actor;
+public final class Identify implements scala.Product, scala.Serializable {
+  static public  akka.actor.Identify apply (Object messageId)  { throw new RuntimeException(); }
+  static public  scala.Option<java.lang.Object> unapply (akka.actor.Identify x$0)  { throw new RuntimeException(); }
+  static private  java.lang.Object readResolve ()  { throw new RuntimeException(); }
+  public  Object messageId ()  { throw new RuntimeException(); }
+  // not preceding
+  public   Identify (Object messageId)  { throw new RuntimeException(); }
+  public  akka.actor.Identify copy (Object messageId)  { throw new RuntimeException(); }
+  public  Object copy$default$1 ()  { throw new RuntimeException(); }
```

Comment:
a nice thing for the future could be to completely filter out Scala stuff that's not intended to be public Java API